app/javascript/pages/country/components/icons/icons-component.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Icons = () => {
+  const svgString = `
+    <svg style="position: absolute; width: 0; height: 0;" width="0" height="0" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
We have this file to store the svg icons: `/app/views/country/_svg_icons.html.erb`